spec/requests/sessions_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe 'Sessions' do
+  it "signs user in and out" do
+    user = create(:user)
+    sign_in user
+    get root_path
+    expect(response).to render_template(:index) 
+    
+    sign_out user
+    get root_path
+    expect(response).not_to render_template(:index) 
```

Comment:
What template are we expecting it to render here?

Reply:
add a test for rending..